Python/Product/Uwp.Interpreter/Uwp.Interpreter.csproj

```diff
@@ -43,6 +43,14 @@
       <Project>{A85D479D-67A9-4BDB-904A-7D86DAF68A6F}</Project>
       <Name>Analysis</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Common\Common.csproj">
```

Comment:
It's for Microsoft.PythonTools.Infrastructure.PathUtils.  Let me know if you want me to stop using it.